dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/SchedulerController.java

```diff
@@ -108,14 +108,14 @@
     @AccessLogAnnotation(ignoreRequestArgs = "loginUser")
     public Result createSchedule(@ApiIgnore @RequestAttribute(value = SESSION_USER) User loginUser,
                                  @ApiParam(name = "projectName", value = "PROJECT_NAME", required = true) @PathVariable String projectName,
```

Comment:
Please replace the projectName with projectCode

Reply:
Thx, i will fix

Reply:
projectCode replaces projectName in Other interface, submit in other PR later